bp_common/syn/Makefile.vcs

```diff
@@ -92,7 +92,7 @@ sim.v: SIM_ERROR  := $(REPORT_DIR)/$(TB).$(CFG).$(TAG).sim.$(SUITE).$(PROG).err
 $(SIM_COLLATERAL): | $(TOUCH_DIR) $(RESULTS_DIR) $(LOG_DIR) $(REPORT_DIR) $(SIM_DIR)
 $(SIM_DIR)/run_simv: | $(SIM_COLLATERAL)
 	cd $(@D); ./simv $(VCS_OPTIONS) $(VCS_PLUSARGS) 2>&1 | tee $(SIM_LOG)
-	-@grep "PASS" $(SIM_LOG) || echo "FAILED" > $(SIM_ERROR)
+	-@grep "FAIL" $(SIM_LOG) && echo "FAILED" > $(SIM_ERROR)
```

Comment:
This is good, but we might also want to add a check for the PASS not being present in the log. Assuming all of our tests will print PASS on success (or multiple PASS if multiple cores), I believe it would be possible for both PASS and FAIL to not be in the log, which should be considered a FAIL for the test.

Reply:
Yeah makes sense. I may add a check for $finish to really be sure

Reply:
Okay, latest commit should be pretty solid. CHecks for PASS & !FAIL & finish